### ISSUE TITLE
508-defect-3 [SCREENREADER]: On this page list should be announced as a list on VoiceOver

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25005]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a VoiceOver user, I want to know how many items there are in the "on this page" navigation list so I can get a quick understanding of the length of the page.

Because of the `usa-unstyled-list` class on the `ul` within "On this page," VoiceOver will not announce the `ul` as a list. This means that VoiceOver users won't hear important cues that other screen reader users will hear including:
- that this is a list "
- how many items there are in the list
- what item they are on out of the total items in the list

## Acceptance Criteria

- [ ] The "On this page" list is announced as a list on VoiceOver

## Additional information

<details>
<summary> Collapsed detail </summary> 
###Environment

VoiceOver, Mac, Safari

### Steps to Recreate

On a Mac in Safari, start up VoiceOver and navigate to the on this page navigation. Tab through the links and confirm it is not announced as a list.

### Proposed Solution 

Adding `role="list"` to the `ul` _should_ do the trick, but it didn't work for me in devtools until I explicitly removed the `usa-unstyled-list` class which isn't normal.

For more, [read Scott's article on `role="list"`](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html)

### Screenshots or Trace Logs

**Before**

https://user-images.githubusercontent.com/14154792/118997495-b3373d80-b956-11eb-8392-0efedc12bcd8.mov

**Expected behavior**
You may notice that I had to remove the utility class here. Hopefully we can find a way to keep the utility class while also keeping the list announcement.

https://user-images.githubusercontent.com/14154792/118997548-bcc0a580-b956-11eb-952c-c9b319ded0c0.mov

</details>

**Comment by mmiddaugh** [comment]
This is a component level defect on a widely used component. 
Michelle will check with Brian/Jeff

**Comment by kelsonic** [comment]
PRs open to add `role="list"` to all `ul` tags that use the class `usa-unstyled-list`:
`content-build`: https://github.com/department-of-veterans-affairs/content-build/pull/410
`vets-website`: https://github.com/department-of-veterans-affairs/vets-website/pull/18082

**Comment by kelsonic** [comment]
Bumped to a 3 due to amount of stakeholders needed during review

**Comment by kelsonic** [comment]
Hey @joshkimux sorry for all the pings but I asked a question on this PR since the `role="list"` for ul tags goes against our linter rules: https://github.com/department-of-veterans-affairs/vets-website/pull/18082#issuecomment-890091224

**Comment by joshkimux** [comment]
@kelsonic this is great! Thank you for the awareness

**Comment by kelsonic** [comment]
PRs merged, going out with the daily deploy today! 🙂

**Comment by brianalloyd** [comment]
@joshkimux can you confirm fix and we'll close this out.  TIA.

**Comment by joshkimux** [comment]
<img width="1196" alt="Screen Shot 2021-08-10 at 9 15 58 AM" src="https://user-images.githubusercontent.com/14154792/128873558-b100fc48-48e6-4d3c-b2e0-6766e09c5242.png">

Great work! It's reading out like a list now 🥳
